pylint/lint/pylinter.py

```diff
@@ -402,7 +402,17 @@ def load_plugin_configuration(self) -> None:
                     "bad-plugin-value", args=(modname, module_or_error), line=0
                 )
             elif hasattr(module_or_error, "load_configuration"):
-                module_or_error.load_configuration(self)
+                # Mypy is not smart enough to realize that we only call
+                # this line if the object has the attr we are looking at.
+                # Hence the one-line type: ignore[union-attr] here.
```

Comment:
You can remove that. The next release of mypy will add better support for `hasattr` at which point we can remove the `type: ignore` as well.

Reply:
I think Marc meant that you can remove the comment explaining the ignore, or at least that would be my suggesiton :smile: 

The comment with the specified code is clear enough in this case as `hasattr` is a known `mypy` issue.

Reply:
Ah, makes sense! Done in the latest commits, thanks both